### PR TITLE
Issue/7 readme

### PR DESCRIPTION
**Pull request by AnnaGsell**
Added Contributions section

Note: new pr because of branch renaming

### CODE REVIEW

**Review by diehmlukas — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
+It includes a stand-alone Android app for fatigue detection and an additional part for data management and further analysis.
+The project is structured as client side (stand-alone app) implemented in Dart/Flutter and a server side (data management) written in Golang.
+Dart/Flutter provides the ability to simply add an iOS, desktop and / or web application besides Android.
+
+
+soTired is a rewrite of the application presented in the [Validation of a Smartphone-Based Approach to In Situ Cognitive Fatigue Assessment](https://mhealth.jmir.org/2017/8/e125) paper by Edward Price, George Moore, Leo Galway and Mark Linden.
+
+## Contributions
+
+For communication throw the code, please use one of the following tags:
```
Comment (carina-str) [inline README.md:15]: What does "throw the code" mean?

Comment (AnnaGsell) [reply README.md:15]: sorry, should be 'through"

File: README.md [hunk 2]
```diff
@@ -1 +1,31 @@
-# soTired
\ No newline at end of file
+# soTired
+
+## Motivation
+
+soTired is an application for cognitive fatigue assessment.
+It includes a stand-alone Android app for fatigue detection and an additional part for data management and further analysis.
+The project is structured as client side (stand-alone app) implemented in Dart/Flutter and a server side (data management) written in Golang.
+Dart/Flutter provides the ability to simply add an iOS, desktop and / or web application besides Android.
```
Comment (mac641) [inline README.md:8]:
```suggestion
Besides Android, Dart/Flutter provides the ability to add an iOS, desktop and / or web application from a single codebase.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,31 @@
...
+## Contributions
+
+For communication through the code, please use one of the following tags:
+```
+FIXME - for a bug which can't be fixed now
+TODO - for a feature or are a design change that has to be done later on
+NOTE - for an important information
+```
+
+The git workflow for this repo is rebase-merge, which results in a history of this form:
```
Comment (mac641) [inline README.md:22]:
```suggestion
The git workflow for this repo is **rebase-merge**, which results in a history of this form:
```

**Review by mac641 — APPROVED** [review]
Very few suggestions for improvement... 😁
But I guess, I'm gonna approve anyway. 👍🏿

**Review by mac641 — APPROVED** [review]
Awesome work :D

**Pull request merged**